### ISSUE TITLE
Cannot bootstrap for development on jekyll-admin.

### ISSUE THREAD

**Issue by derek-adair** [jekyll/jekyll-admin#704]
### Description:
./scripts/bootstrap doesn't work for me.  Depending on the version of node I use to build, i get various errors from yarn install.

### Tell us a bit about yourself:
* Version of JekyllAdmin I'm using:  [origin/master](https://github.com/jekyll/jekyll-admin/tree/master)
* Version of Jekyll I'm using: 4.3.1
* Version of NodeJS I'm using: Tried 16.x and 18.x
* Operating System Ubuntu 20.04

### Steps to reproduce:
1. git clone git@github.com:jekyll/jekyll-admin.git
2. ./scripts/bootstrap

### I expected the following:
A local copy of jekyll-admin I can modify.

### But got the following, instead:
Various yarn errors depending on the version of node i'm using, I can supply them if necessary but I dont think this is the case.

### Other details:
the frontend dependencies have not been updated in >3 years.  It would be prudent to regenerate your lock files.  Additionally, it can be troublesome to commit package-lock.json and yarn.lock.  These were compiled on node 16, which is no longer supported.

**Comment by derek-adair** [comment]
removing the lock files and running ./scripts/bootstrap w/ node 18 works.

**Comment by derek-adair** [comment]
[These things make this project a **MASSIVE** security risk](https://stackoverflow.com/a/73027407/231435).

**Comment by thaleshcv** [comment]
Hi @derek-adair any progress with this issue?

**Comment by derek-adair** [comment]
Nope.  No longer using this project for various reasons which are far beyond the scope of this issue.  I still recommend using jekyll for any developer but i've moved over to Ghost for clients that need a UI for CMS.

IMO This is still a vital issue in order for this project to survive so someone should probably submit a PR.

**Comment by cynic64** [comment]
`scripts/bootstrap`, `scripts/build`, and `scripts/server-frontend` all work with node 13 as long as you don't delete `package-lock.json`. But I agree it would be nice to use newer versions.
